ribs/emitters/_evolution_strategy_emitter.py

```diff
@@ -85,7 +87,8 @@ def __init__(self,
             raise ValueError(f"Invalid selection_rule {selection_rule}")
         self._selection_rule = selection_rule
 
-        if restart_rule not in ["basic", "no_improvement"]:
+        if restart_rule not in ["basic", "no_improvement"] and isinstance(
+                restart_rule, int):
```

Comment:
Also check for `np.integer`, i.e. `isinstance(restart_rule, (int, np.integer))`. Do this in `_check_restart` too